colossalai/shardformer/policies/bloom.py

```diff
@@ -24,12 +24,12 @@
 class BloomPolicy(Policy):
     def __init__(self) -> None:
         super().__init__()
-        import transformers
-        from packaging.version import Version
+        # import transformers
+        # from packaging.version import Version
 
-        assert Version(transformers.__version__) <= Version(
-            "4.33.0"
-        ), "The Bloom model should run on a transformers version not greater than 4.33.0."
+        # assert Version(transformers.__version__) <= Version(
+        #     "4.33.0"
+        # ), "The Bloom model should run on a transformers version not greater than 4.33.0."
```

Comment:
You could just remove these lines (remove these comments). For your reference, the assertion has been removed in main